3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.iOS/Info.plist

```diff
@@ -23,8 +23,10 @@
 	<string>1.0</string>
 	<key>IntuneMAMSettings</key>
 	<dict>
+		<key>ADALAuthority</key>
+		<string>https://login.microsoftonline.com/TODO -tenantId</string> <!--For single tenant app replace organizations with your tenant ID or name. Remove entry for others -->
```

Comment:
Comment references "organizations" string which doesn't exist. Perhaps we should just use the organizations authority and instruct devs to change it if their app is single-tenant so that multi-tenant apps work by default?

Reply:
Good suggestion. Fixed